model/simple/taskbehavior.go

```diff
@@ -235,11 +240,14 @@ func (tb *TaskBehavior) Done(ctx model.TaskContext) (notifyFlow bool, taskEntrie
 		//Otherwise branch while no expression link to follow
 		if exprOtherwiseLinkInst != nil && hasExprLink && !exprLinkFollowed {
 			exprOtherwiseLinkInst.SetStatus(model.LinkStatusTrue)
+			exprOtherwiseTaskEntry.EnterCode = 3
```

Comment:
Expression otherwise is a true link so should we change the Entercode to 1 or 2?